spec/super_good/solidus_taxjar/backfill_transactions_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "spec_helper"
+
+RSpec.describe SuperGood::SolidusTaxjar::BackfillTransactions do
+  describe ".call" do
+    subject { described_class.call }
+
+    let!(:complete_order) { create :completed_order_with_totals }
+    let(:api_spy) { instance_spy(::SuperGood::SolidusTaxjar::Api) }
+
+    around do |example|
+      ::SuperGood::SolidusTaxjar.test_mode = true
+      example.run
+      ::SuperGood::SolidusTaxjar.test_mode = false
+    end
+
+    before do
+      reported_order = create :completed_order_with_totals
+      create(:taxjar_order_transaction, order: reported_order)
+      create :order
+
+      allow(::SuperGood::SolidusTaxjar).to receive(:api).and_return(api_spy)
```

Comment:
We could get rid of this if we injected the dependency instead of having it be implicit.

Reply:
Updated.